### PR TITLE
Add C# DEM.Net

### PR DESCRIPTION
**Pull request by xfischer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,6 +290,7 @@ To compare WebGL-based glTF loaders, see [gltf-test](https://github.com/cx20/glt
 | [Aspose.3D for .NET](http://www.aspose.com/products/3d/net) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat)| Import, export, and convert glTF |
 | [MixedRealityToolkit-Unity](https://github.com/Microsoft/MixedRealityToolkit-Unity) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat) | A collection of scripts and components intended to accelerate development of applications targeting Microsoft HoloLens and Windows Mixed Reality headsets. |
 | [SharpGLTF](https://github.com/VPenades/SharpGLTF) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat) | C# toolkit for reading and writing glTF files. |
+| [DEM.Net](https://github.com/dem-net/dem-net) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat) | C# toolkit for Digital Elevation Models, exporting models with textures and normal maps to glTF files. |
```
Comment (bghgary) [inline README.md]: This link doesn't work. Did you mean this?

```suggestion
| [DEM.Net](https://github.com/dem-net/DEM.Net) | ![status](https://img.shields.io/badge/glTF-2%2E0-green.svg?style=flat) | C# toolkit for Digital Elevation Models, exporting models with textures and normal maps to glTF files. |
```

Comment (xfischer) [reply README.md]: Yes @bghgary ! Sorry and thank you !

Comment (bghgary) [reply README.md]: Did you want to commit the suggestion? I noticed you marked this as resolved.

**Comment by xfischer** [comment]
Yes ! I did not catch the commit button :)

**Pull request merged**